src/main/java/com/hotjava/app/hotjava/dto/Vehicle.java

```diff
@@ -4,13 +4,12 @@
 
 public @Data
 class Vehicle {
-    private int vehicleSubmissionID;
-    private String vehicleOwnerName;
-    private String vehicleDescription;
-    private String vehicleYear;
-    private String vehicleMake;
-    private String vehicleModel;
-    private String vehicleScore;
-
+    private int SubmissionID;
+    private String OwnerName;
+    private String Description;
+    private String Year;
+    private String Make;
+    private String Model;
+    private String Score;
```

Comment:
Removed redundant 'vehicle' from variables within vehicle object. While this is personal preference, I found it easiest to read and understand when I did not have to look at the jumbled 'vehicles'.

Reply:
I agree that it is redundant, but the result is still not ideal.

Variable names in Java should be lowerCamelCase.

Reply:
implemented with Camel Case:
https://github.com/coffee-ns/hotjava/commit/b60898f3141483b2a7f2d82b1df61f3563f860b5